### PR TITLE
Login adapters and base readme

### PR DESCRIPTION
**Pull request by atStarling**
Additions
- Wrote the readme (still needs some parts to be filled by others)
- Login Interface adapters (might be overwritten by Pooja's branch pull request)

### CODE REVIEW

File: README.md
Comment (matt-dahlgren) [inline README.md:1]: README is looking good, we have to (obviously) keep adding and contributing to this once our program is more functional, but of what we have so far your explanations are very comprehensible. In Images I think we should show: a FreePlay Game Board, the AssistedMode Game board, the multiplayer gameboard and the end of game report!

Also, we need to add the API that we used for the mailing!

File: src/main/java/use_case/login/LoginInteractor.java
Comment (matt-dahlgren) [inline src/main/java/use_case/login/LoginInteractor.java:1]: The logic of the interactor is starting to look great, only thing I would add would be to add some comments in the if and else statements  of the execute function as to make the function a bit more readable and easier to understand.

File: src/main/java/use_case/login/LoginUserDataAccessInterface.java
Comment (matt-dahlgren) [inline src/main/java/use_case/login/LoginUserDataAccessInterface.java:1]: This interface looks amazing! Javadocs are well documented and the style is very consistent.

**Review by matt-dahlgren — APPROVED** [review]
Looks very good and I would merge with main with the caveat of looking over to make the code a bit more readable later.

File: src/main/java/use_case/login/LoginUserDataAccessInterface.java
Comment (atStarling) [reply src/main/java/use_case/login/LoginUserDataAccessInterface.java:1]: Thank you!

**Pull request merged**